### ISSUE TITLE
Add support to detect blueprint references in the Binding Editor

### ISSUE THREAD

**Issue by jdaignault** [DoubleDeez/MDFastBinding#26]
It would be very useful to be able to detect references to blueprint properties and functions that exist in bindings. As it stands now, only references within the Event Graph and Blueprints are picked up, which prevents references in the Binding Editor from being discovered when using editor search features such as **Find References**, **Find In Blueprints**, and the **Search** tool. Additionally, those references are not picked up by the Rider for Unreal Engine IDE.

![image](https://github.com/DoubleDeez/MDFastBinding/assets/1145663/40b84ed0-a95b-4236-bda3-0f368caaa941)

![image](https://github.com/DoubleDeez/MDFastBinding/assets/1145663/d192b8e4-2a98-40df-9157-194ef9a09d43)

**Comment by DoubleDeez** [comment]
Does the Reference Viewer work for asset references?

The search tool needs to be changed to properly support multiple extensions per asset type, it's on my backlog to PR that.

**Comment by jdaignault** [comment]
Yes-ish. See issue #21, asset references are established but only when they're populated in a certain way.
